ir/src/main/scala/org/scalajs/core/ir/Definitions.scala

```diff
@@ -66,6 +66,9 @@ object Definitions {
   val AncestorsOfPseudoArrayClass = Set(
       ObjectClass, SerializableClass, CloneableClass)
 
+  /** Name of the static initializer method. */
+  final val StaticInitializerName = "clinit___"
```

Comment:
Yes. Just like the instance constructor is named `<init>` on the JVM, the static initializer is named `<clinit>` (for class init)